include/libopencm3/stm32/f0/rcc.h

```diff
@@ -244,6 +244,7 @@ Control</b>
 #define RCC_AHBENR_CRCEN			(1 << 6)
 #define RCC_AHBENR_FLTFEN			(1 << 4)
 #define RCC_AHBENR_SRAMEN			(1 << 2)
+#define RCC_AHBENR_DMA2EN			(1 << 1)
```

Comment:
You also need to add it to enums  down at https://github.com/libopencm3/libopencm3/blob/a9c0008290ae7026fd4bc5c662966c6d88e80697/include/libopencm3/stm32/f0/rcc.h#L413 please

Reply:
Isn't it [already there](https://github.com/libopencm3/libopencm3/blob/a9c0008290ae7026fd4bc5c662966c6d88e80697/include/libopencm3/stm32/f0/rcc.h#L416)?

Reply:
true, looks like someone added that, and just didn't add the rest of it in 54c6c833f ok